File: src/Planning/Kick.java
package Planning;
import org.zeromq.ZMQ;
import org.zeromq.ZMQ.Context;
import org.zeromq.ZMQ.Socket;

public class Kick {


private static Context context;
private static Socket socket;

public static void main(String args[]) throws InterruptedException{
	context = ZMQ.context(1);
	socket = context.socket(ZMQ.REQ);
    socket.connect("tcp://127.0.0.1:5555");
    	
}

}

File: src/BluetoothServer/kick.py
import sys
import time

import zmq

context = zmq.Context()

print "Connecting to server..."
socket = context.socket(zmq.REQ)
socket.connect("tcp://127.0.0.1:5555")
print "Connected!"

# Kick!
#socket.send ("1 0 0 0 0")

while 1:
    #socket.send ("1 1 2 3 4")
    socket.send("3")

    print socket.recv()
    
    
    time.sleep(5)